prod/westeurope/internal/api/apim/api_management/terragrunt.hcl

```diff
@@ -65,7 +61,6 @@ inputs = {
 
   named_values_map = {
     io-functions-admin-url    = "http://${dependency.functions_admin.outputs.default_hostname}"
-    io-functions-services-url = "http://${dependency.functions_services.outputs.default_hostname}"
```

Comment:
You must also remove io-functions-services-key from secrets

Reply:
Yep, done .... thank you.